src/main/scala/com/timeout/kamon/cloudwatch/KamonCloudWatchExtension.scala

```diff
@@ -0,0 +1,30 @@
+package com.timeout.kamon.cloudwatch
+
+import akka.actor.{ActorRef, ExtendedActorSystem}
+import akka.event.Logging
+import com.timeout.kamon.cloudwatch.KamonSettings.subscriptions
+import com.typesafe.config.ConfigList
+
+import scala.collection.JavaConverters._
+import kamon.Kamon
+
+
+class KamonCloudWatchExtension(system: ExtendedActorSystem) extends Kamon.Extension {
+  val log = Logging(system, classOf[KamonCloudWatchExtension])
+  log.info("Starting the Kamon CloudWatch extension")
+
+  implicit val ec = system.dispatcher
+
+  val shipper: ActorRef = system.actorOf(MetricsShipper.props, "cloudwatch-metrics-shipper")
+  val subscriber: ActorRef = system.actorOf(MetricsLogger.props(shipper), "cloudwatch-metrics-logger")
+
+  subscriptions.entrySet.asScala.foreach { entry =>
```

Comment:
Nitpick: it feels like most of the Java to Scala gymnastic should be done in `KamonSettings` instead